packages/cli/src/editor/helpers/use-menu-structure.tsx

```diff
@@ -151,6 +151,34 @@ export const useMenuStructure = (closeMenu: () => void) => {
 						subMenu: null,
 						quickSwitcherLabel: 'New still...',
 					},
+					{
+						id: 'open-in-editor',
+						value: 'open-in-editor',
+						label: 'Open in code editor...',
```

Comment:
Can we rename it to `Open in ${window.remotion_editorName}`?
The `...` signals that a modal is opened in the Preview, so I would remove the `...`.